extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -214,9 +214,21 @@ class UpdateReplicationStatus extends BackbeatTask {
             entry = this._getNFSUpdatedSourceEntry(sourceEntry, refreshedEntry);
         } else if (newStatus === 'COMPLETED') {
             entry = refreshedEntry.toCompletedEntry(site);
+
+            let replayCount = 0;
             if (sourceEntry.getReplayCount() >= 0) {
-                this.metricsHandler.replaySuccess();
+                replayCount = this.replayTopics.length - sourceEntry.getReplayCount();
+                this.metricsHandler.replaySuccess({ location: site, replayCount });
             }
+
+            const labels = {
+                location: site,
+                replayCount,
+                replicationStatus: 'COMPLETED',
```

Comment:
If we reuse such statuses, we could create constants, it will be easier to manage

Reply:
i totally agree, `const` (or better yet a typescript `enum`) would be better... but these should actually be part of the metadata models (e.g. in arsenal), and are used much more widely than just here: so i will leave that for another day